packages/core/src/kernelTemplate.js

```diff
@@ -1,5 +1,16 @@
 // LavaMoat Prelude
 (function () {
+  const globalRef = globalThis;
+
+  const {
+    includeSnow,
+  } = __lavamoatBuilderOptions__
+
+  const SNOW = globalRef.SNOW;
+  if (!SNOW && includeSnow) {
+    throw 'LavaMoat - Snow is expected to exist but it does not'
```

Comment:
```suggestion
    throw new Error('LavaMoat - Snow is expected to exist but it does not');
```
?
Or if there's a motivation behind throwing a string, an explanatory comment would be great

Reply:
throwing a string means there's no stack trace. I'd recommend against it unless there's an important security reason.

Is this an attempt to tame errors by not using Errors?